lib/turbo_tests/reporter.rb

```diff
@@ -43,11 +46,18 @@ def add(name, outputs)
           when "d", "documentation"
             RSpec::Core::Formatters::DocumentationFormatter
           else
+            custom_formatters = true
             Kernel.const_get(name)
           end
 
         @formatters << formatter_class.new(output)
       end
+
+      start if custom_formatters
```

Comment:
Is it safe to call `start` even without `custom_formatters`?

```suggestion
      start
```

Reply:
It's safe but unnecessary : the `start` method computes the examples count which you don't need on default formatters.

On large applications, it could take few seconds so I had the condition to preserve performances on default cases:

```bash
$ time bundle exec rspec --dry-run --format json --out=test.json                                                                                                                                                                              
real 11.78
user 8.20
sys 2.27
```
(on a 38196 LOC app)

Reply:
Makes sense, thank you!